### PR TITLE
media: Prevent memcpy size can be a negative value

### PR DESCRIPTION
**Pull request by jarvisjang**
Fix SVACE : 217935

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [068bedea5d236409121fd0119494d940348c0b40] - Code Rule Check Result:
framework/src/media/audio/audio_manager.c:595: ERROR: [BRC_M_SMT] else should follow close brace '}'

**Comment by seinfra** [comment]
Target : [068bedea5d236409121fd0119494d940348c0b40] - Code Rule Check (C++) OK.

File: framework/src/media/audio/audio_manager.c [hunk 1]
```diff
@@ -584,11 +584,18 @@ int start_audio_stream_in(void *data, uint32_t frames)
...
+				ret = get_input_frames_byte_size(srcData.output_frames_gen);
+				if (ret > 0) {
+					memcpy(data + get_input_frames_byte_size(frames_copied), srcData.data_out, ret);
+					frames_copied += srcData.output_frames_gen;
+					frames_used += srcData.input_frames_used;
 
-				medvdbg("Record resampled %d/%d\n", frames_used, frames_total);
+					medvdbg("Record resampled %d/%d\n", frames_used, frames_total);
+				}
+				else {
```
Comment (sunghan-chang) [inline framework/src/media/audio/audio_manager.c]: You must put else at same line of closing bracket.
```
} else {
```

Comment (jarvisjang) [reply framework/src/media/audio/audio_manager.c]: Fixed

**Comment by seinfra** [comment]
Target : [54255246cd8791e34fe9bc3c394be866914cba59] -

**Comment by seinfra** [comment]
Target : [54255246cd8791e34fe9bc3c394be866914cba59] - Code Rule Check (C++) OK.

**Pull request merged**